requirements.txt

```diff
@@ -4,7 +4,6 @@ numpy==1.17.3
 
 # Commons requirements
 jsonschema==3.1.1
-numpy==1.17.3
```

Comment:
why removing numpy requirement ? numpy is used in this lib

Reply:
There was 2 `numpy==1.17.3` lines in this file.